tests/unit/amo/components/TestAddonsByAuthorsCard.js

```diff
@@ -347,6 +350,26 @@ describe(__filename, () => {
     sinon.assert.notCalled(dispatchSpy);
   });
 
+  it('should have extensions length equal to numberOfAddons props', () => {
+    const root = renderAddonsWithType({
+      addonType: ADDON_TYPE_EXTENSION,
+      multipleAuthors: false,
+      numberOfAddons: 4,
+    });
+
+    expect(root.find(AddonsCard).props().addons).toHaveLength(4);
```

Comment:
If `4` is related to `numberOfAddons` above, I would create a variable for that.